.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+
+version: 2
+jobs:
+  build:
+    working_directory: ~/graft
+    parallelism: 1
+    docker:
+    - image: circleci/node:8.11.4-stretch
```

Comment:
Minor: If we don’t use `node` here, we could use the FP Complete Stack Docker image: `fpco/stack-build`

See: https://github.com/zoomhub/zoomhub/blob/d67c750a1036427cc9bd4dc8830ba74dc52701d5/.circleci/config.yml#L27

Reply:
Yes, we don't use Node. I was looking around for a better image and couldn't find one, so I used the same we already use in MyCV.

I'll try out the FPComplete's one. Does it have the latest stack always there?

Reply:
Good question. Looks like you can narrow it down by tag: https://hub.docker.com/r/fpco/stack-build/tags/

I should do that for my own project to match the LTS to be what we use inside the project.

Reply:
I'm not sure what exactly is the difference between tags, but it sure looks like these images don't even have GHC installed - judging by the [build log](https://circleci.com/gh/collegevine/graft/25).